src/opm/io/eclipse/EclUtil.cpp

```diff
@@ -175,6 +183,11 @@ uint64_t Opm::EclIO::sizeOnDiskBinary(int64_t num, Opm::EclIO::eclArrType arrTyp
         if (num > 0) {
             auto sizeData = Opm::EclIO::block_size_data_binary(arrType);
 
+            if (arrType == Opm::EclIO::C0NN){
+                std::get<1>(sizeData)= std::get<1>(sizeData) / std::get<0>(sizeData) * elementSize;
```

Comment:
I understand that it is OK - but assigning to `std::get<>()` looked weird - would it be possible to use the underlying variables here instead?

Reply:
I agree.  Hopefully this is better.

```c++
    int sizeOfElement = std::get<0>(sizeData);
    int maxBlockSize = std::get<1>(sizeData);
    int maxNumberOfElements = maxBlockSize / sizeOfElement;

    if (arrType == Opm::EclIO::C0NN){
            maxBlockSize= maxBlockSize / sizeOfElement * elementSize;
            sizeOfElement = elementSize;
     }
```

Reply:
I think this is much better!